codereview.java

```diff
@@ -0,0 +1,36 @@
+public class RuntimeIssuesDemo {
+
+    public static void main(String[] args) {
+        int[] numbers = {1, 2, 3};
+        System.out.println(numbers[3]); 
+
+        
+        String str = null;
+        System.out.println(str.length()); 
```

Comment:
<b>code: System.out.println(str.length()); - </b>
NullPointerException might occur due to calling length() on a null reference